lib/exercism/team_stream_filters.rb

```diff
@@ -72,7 +72,7 @@ def sql
     # rubocop:disable Metrics/MethodLength
     def views_sql
       <<-SQL
-      SELECT 1 AS id, COUNT(views.id) AS total
+      SELECT 'team_stream' AS id, COUNT(1) AS total
```

Comment:
Exactly! It is only needed in [this](https://github.com/exercism/exercism.io/blob/master/lib/exercism/track_stream_filters.rb#L10) step.